src/components/Editor/ItemEditor.vue

```diff
@@ -779,17 +777,12 @@ export default {
         this.selectedItemDetail.max_char_limit = value;
       },
     },
-    isThisSurveyQuestion: {
+    isSelectedItemSurveyQuestion: {
       get() {
         // extract whether question is for survey
         if (this.selectedItemDetail == null) return false;
         return this.selectedItemDetail.survey;
       },
-      set(value) {
-        // set whether question is for survey
-        this.isSurveyQuestion = true;
-        this.selectedItemDetail.survey = value;
-      },
     },
```

Comment:
You removed the `set()` method right? We don't need the `get()` method explicitly written.
See all the other computed variables, the `get` is implicit / implied.

Reply:
but then 
![image](https://user-images.githubusercontent.com/73693628/155107337-33b20654-1571-45bb-b54c-8a5136223032.png)
it is showing this